Projektskabelon-master/src/main/java/FunctionLayer/OverlayMaterialCalculator.java

```diff
@@ -142,6 +144,7 @@ public  static Material screwForOverlayOneWall ( Wall wall, String overlayName)
         overlay.setSize(3600);
         overlay.setAmount(quantity);
         overlay.setComment("Beklædning");
+        overlay.setPrice(LogicFacade.getPrice(overlay.getId()));
```

Comment:
If you set the price here, it will be calculated for the needed size, not for the available one. I calculate de available sizes first in function layer